src/boards/rainbow_esp.h

```diff
@@ -57,22 +60,37 @@ class BrokeStudioFirmware: public EspFirmware {
 		FILE_WRITE,
 		FILE_APPEND,
 		GET_FILE_LIST,
-		GET_FREE_FILE_ID,
+		GET_FREE_FILE_ID, //TODO
 	};
 
 	// Defined message types from ESP to CPU
 	enum class e2n_cmds_t : uint8 {
 		READY,
+
 		FILE_EXISTS,
+		FILE_DELETE,
 		FILE_LIST,
 		FILE_DATA,
 		FILE_ID,
+
 		WIFI_STATUS,
 		SERVER_STATUS,
 		HOST_SETTINGS,
+
 		RND_BYTE,
 		RND_WORD,
+
 		MESSAGE_FROM_SERVER,
+
+		GET_N_BYTES = 253,
+
+		SUCCESS = 254,
+		FAILURE = 255 // ERROR throw an error here...
```

Comment:
If I replace FAILURE with ERROR, I can't compile.

Reply:
It builds flawlessly on my side when renaming it ERROR. Possibly some clash with a macro name on your system.

I am still not convinced that this comment is useful (it indicates a very slight variation from the doc). Also you should add a comma after the entry (other enums of this file all have a trailing comma after the last entry).

Reply:
It seems you really want this comment. No problem, but be crystal clear in the phrasing.

Here is my proposal:
`FAILURE = 255, // Equivalent to the value ERROR in Rainbow's documentation, renamed FAILURE to avoid compilation issue`

It explains the **what** and the **why** while avoiding the word "throw" that has a meaning in C++, related to runtime errors, not compilation errors.

Also, I would be curious to see the compilation error. I succeed to compile the Linux version, so there may be a way to work around the compilation issue on Windows.